mypy/build.py

```diff
@@ -294,7 +298,9 @@ def default_lib_path(data_dir: str,
     # Add fallback path that can be used if we have a broken installation.
     if sys.platform != 'win32':
         path.append('/usr/local/lib/mypy')
-
+    assert path, "Could not resolve typeshed subdirectories. If you are using MyPy " \
```

Comment:
Heh, I forgot about it, but indeed we dropped support for 3.2. (And may it R.I.P. :-)

Regarding the error message, the Error instance hasn't even been constructed at the point this function is called, so the best we could do would be to `print(message, file=sys.stderr)`and then `sys.exit(1)`. Which isn't terrible, main.py even has a helper that does just that. (Though I think eventually we should refactor the fatal/blocking error handling; I've also got the idea that using sys.exit(2) for all fatal errors would be preferable.)